app/src/main/java/mod/hey/studios/activity/managers/java/ManageJavaActivity.java

```diff
@@ -144,13 +174,41 @@ private String getCurrentPkgName() {
         }
     }
 
+    private RadioButton createRadioButton(CharSequence text, int id) {
+        RadioButton r =  new RadioButton(this);
+        r.setText(text);
+        r.setId(id);
+
+        return r;
+    }
+
     private void showCreateDialog() {
         final AlertDialog dialog = new AlertDialog.Builder(this).create();
         View root = getLayoutInflater().inflate(Resources.layout.dialog_create_new_file_layout, null);
 
         final EditText inputName = root.findViewById(Resources.id.dialog_edittext_name);
         final RadioGroup radio_fileType = root.findViewById(Resources.id.dialog_radio_filetype);
 
+        ((TextView)root.findViewById(Resources.id.dialog_radio_filetype_class)).setText("Java Class");
+        ((TextView)root.findViewById(Resources.id.dialog_radio_filetype_activity)).setText("Java Activity");
+
+        // Literally the worst way of doing this but editing resources.arsc is much more painful
+        final int dialog_radio_filetype_kt_class = 1001;
+        final int dialog_radio_filetype_kt_activity = 1001 + 1;
+        {
```

Comment:
When 1002?

Reply:
I probably meant to do `dialog_radio_filetype_kt_class + 1` but yeah you can just change it to `1002`.

Or, better yet, you guys can delete the manual `RadioButton` creating logic and do it in XML since Sketchware Pro can now be compiled directly from Android Studio.

Reply:
Doing in XML would be much better.